shared/attributes.asciidoc

```diff
@@ -154,4 +154,6 @@
 :api-response-codes-title: Response codes
 :api-response-body-title:  Response body
 :api-example-title:        Example
```

Comment:
I think we should consider axing the `api-example-title` attribute or just changing its value to `Examples`.

Not a big deal though. That might be better as a separate PR.

Reply:
I think we shouldn't remove it (yet), since I have some pages that are using it. I agree that it likely makes sense to change it to "Examples" eventually but for now I think it's fine as-is, since the templates will no longer mention it.